src/main.rs

```diff
@@ -162,17 +162,13 @@ impl RsIdentify {
     #[allow(non_snake_case)]
     fn dscheck_NoCloud(&self) -> bool {
         // TEST GAP: nocloud and nocloud-net are not tested for both writable and regular paths
-        for seed_type in &["nocloud", "nocloud-net"] {
-            if self.seed_path_exists(None, seed_type, "user-data")
-                && self.seed_path_exists(None, seed_type, "meta-data")
-            {
-                return true;
-            }
-
-            if self.seed_path_exists(Some("writable/system-data"), seed_type, "user-data")
-                && self.seed_path_exists(Some("writable/system-data"), seed_type, "meta-data")
-            {
-                return true;
+        for prefix in &[None, Some("writable/system-data")] {
```

Comment:
this isn't a real path, right?

Reply:
Yep, on Ubuntu Core images the root filesystem is immutable (and identical across all devices); this is a writable path that's mounted for configuration.